scoap3/modules/frontpage/templates/scoap3_frontpage/home.html

```diff
@@ -54,7 +54,7 @@ <h3 class="text-left">Browse by journal:</h3>
               </ul>
             </div>
             <div class="tab-pane" id="2">
-              <h3>Participating countries:</h3>
+              <h3></h3>
```

Comment:
Same here

Reply:
leaving ''\<h3>'' empty is just because of styling, not to have 0 gap between tab and list